admin/tools/docker/useful-cmd.sh

```diff
@@ -1,11 +1,11 @@
 # These commands require Docker on your machine and that you belong to 'docker' group
```

Comment:
I believe this file is really documentation -- is there a better place for it to live?